### PR TITLE
Interactive

### PR DESCRIPTION
**Pull request by Farnazmdi**
I checked the code in my local, it runs. 
In each cell we can manipulate the 4 Hill parameters `[max_g1_prog, max_g2_prog, max_g1_death, max_g2_death]`.
I will play with it a little bit tomorrow and will update you if I found something promising. We could look at that together, too.

### CODE REVIEW

File: src/DrugResponseModel.jl [hunk 1]
```diff
@@ -16,6 +16,7 @@ import ExponentialUtilities
 import LineSearches
 using Polynomials
 using DiffResults
+using Interact
```
Comment (aarmey) [inline src/DrugResponseModel.jl:19]: If Interact isn't used within the package then it doesn't need to be imported or placed in the package manifest.

**Review by aarmey — CHANGES_REQUESTED** [review]
One change to make sure this builds.

File: src/DrugResponseModel.jl (hunk 1 above)
Comment (Farnazmdi) [reply src/DrugResponseModel.jl]: Should I remove changes made to the manifest.toml, too?

Comment (aarmey) [reply src/DrugResponseModel.jl]: What is the purpose of the manifest?

Comment (Farnazmdi) [reply src/DrugResponseModel.jl]: It has information about dependencies. So I'll remove the changes.

**Pull request merged**